src/domains/wallet/pages/CreateWallet/CreateWallet.test.tsx

```diff
@@ -32,36 +62,53 @@ describe("CreateWallet", () => {
 		expect(selectAssetsInput).toBeTruthy();
 
 		await act(async () => {
-			fireEvent.change(selectAssetsInput, { target: { value: "Bitco" } });
+			fireEvent.change(selectAssetsInput, { target: { value: "ARK" } });
 		});
 
 		await act(async () => {
 			fireEvent.keyDown(selectAssetsInput, { key: "Enter", code: 13 });
 		});
 
-		expect(getByTestId("select-asset__selected-Bitcoin")).toBeTruthy();
+		expect(getByTestId("select-asset__selected-ARK - Mainnet")).toBeTruthy();
```

Comment:
IDs and Classes shouldn't contain whitespace characters.

Reply:
@alexbarnsley I will create a separate PR to fix this in `SelectNetwork`, so I can update my PR as well.

Reply:
thanks @brenopolanski 👌 I agree it shouldn't contain whitespace